static/html/index.html

```diff
@@ -18,12 +18,24 @@
     <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.2.1/jquery.min.js"></script>
     <script type="application/javascript">
       $(document).ready(function() {
-        var timeout = {{.Retry}}*1000
 
         function showStatus(header, body) {
           $("#header").text(header)
           $("#status").text(body)
         }
+
+        function expBackoffInterval(timeout, minRetry) {
+          var current = timeout * 2
+          return function next() {
+            current = Math.max(current /2, minRetry)
+            return current
+          }
+        }
+
+        var initialRetry = {{.InitialRetry}} * 1000
+        var minRetry = {{.MinRetry}} * 1000
```

Comment:
initialRetry and minRetry both are time. They should be renamed to something more appropriate which gives out this information.

Reply:
ack.